src/Tribe/Main.php

```diff
@@ -364,38 +369,80 @@ public function maybe_set_common_lib_info() {
 		}
 
 		/**
-		 * Plugins shouldn't include their functions before `plugins_loaded` because this will allow
-		 * better compatibility with the autoloader methods.
+		 * Resets the global common info back to ET's common path
 		 *
-		 * @return void
+		 * @since 4.9.3.2
 		 */
-		public function plugins_loaded() {
+		private function reset_common_lib_info_back_to_et() {
+			$et = Tribe__Tickets__Main::instance();
+			$et_common_version = file_get_contents( $et->plugin_path . 'common/src/Tribe/Main.php' );
+			$GLOBALS['tribe-common-info'] = [
+				'dir'     => "{$et->plugin_path}common/src/Tribe",
+				'version' => $et_common_version,
```

Comment:
(2/2) ...and are then setting the version to be the source. Shouldn't we be parsing out the version number?